index.js

```diff
@@ -17,43 +19,40 @@ async function run() {
       repository
     } = github.context.payload;
 
-    const env = core.getInput('env');
-    const path = core.getInput('path');
+    const environment = core.getInput('ENVIRONMENT');
+    const path = core.getInput('PATH');
 
-    if (env !== 'production' && env !== 'staging') {
-      throw new Error('Environment input must be production or staging.');
+    if (environment !== 'production' && environment !== 'staging') {
+      throw new Error('Environment input must be provided (production and staging).');
```

Comment:
It should be production **or** staging, no?

Reply:
solved!

Reply:
The && operator is correct! I was only talking about the error message:
**(production and staging).** to **(production or staging).**

Reply:
hmmmm ok